Makefile

```diff
@@ -40,7 +40,10 @@ bdd-test: build-vct-docker
 .PHONY: build-vct
 build-vct:
 	@echo "Building verifiable credentials transparency (vct)"
-	@go build -o build/bin/vct cmd/vct/main.go
+	@CGO_ENABLED=0 GOOS=linux go build -o build/dist/bin/vct-linux cmd/vct/main.go
+	@CGO_ENABLED=0 GOOS=darwin go build -o build/dist/bin/vct-darwin cmd/vct/main.go
```

Comment:
This is the amd64 darwin build. Should explicitly add GOARCH=amd64 and include in the binary name.

Reply:
Also arm64 is also important for darwin.